app/assets/sass/modules/_web-payments.scss

```diff
@@ -1,20 +1,35 @@
-#apple-pay-container,
-#google-pay-container {
+.apple-pay-container{
   display: none;
-}
 
-// Toggle Apple Pay
-.apple-pay-available {
-  #apple-pay-container {
+  .apple-pay-available & {
```

Comment:
Parent selector - only show this when a parent element has `apple-pay-available` on it